src/main/java/org/everit/osgi/dev/maven/AbstractEOSGiMojo.java

```diff
@@ -55,12 +74,38 @@
   @Parameter(property = "executedProject")
   protected MavenProject executedProject;
 
+  private GoogleAnalyticsTrackingService googleAnalyticsTrackingService;
+
+  @Parameter(defaultValue = "${mojoExecution}", readonly = true)
+  protected MojoExecution mojo;
+
   /**
    * The Maven project.
    */
   @Parameter(property = "project")
   protected MavenProject project;
 
+  /**
+   * Skip tracking or not. That means send event statistics to Google Analytics or not. Default
+   * value is <code>false</code> that means send statistics.
+   */
+  @Parameter(property = "eosgi.tracking.skip", defaultValue = "false")
```

Comment:
Should be "eosgi.analytics.skip" to match the naming of the other parameters: eosgi.analytics.referer and eosgi.analytics.waiting.time.